.travis.yml

```diff
@@ -1,20 +1,18 @@
 sudo: false
 language: python
 env:
-- DJANGO="django>1.8,<1.9" DJANGO_SETTINGS_MODULE=representatives_votes.tests.settings
+- DJANGO="django>1.8,<1.9" DJANGO_SETTINGS_MODULE=test.settings
 python:
 - "2.7"
-before_install:
-- pip install codecov
 install:
-- pip install $DJANGO pep8 flake8 pytest-django pytest-cov codecov
+- pip install $DJANGO pep8 flake8 codecov
 - pip install https://github.com/political-memory/django-representatives/archive/parltrack.tar.gz#egg=django-representatives
 - pip install -e .
 script:
 - django-admin migrate
 - flake8 representatives_votes/ --exclude migrations --ignore E128
-- py.test
-- cat representatives_votes/contrib/parltrack/tests/dossiers_fixture.json | parltrack_import_dossiers
-- cat representatives_votes/contrib/parltrack/tests/votes_fixture.json | parltrack_import_votes
```

Comment:
Hmmm I don't quite see why you ask that question, but this PR does not really change the way parltrack and the app are coupled, I even tried to decouple it more (by adding JSON data sources).

Those lines you're commenting about are just getting rid of unneeded tests (those are run by `django-admin test` now).